runelite-client/src/main/java/net/runelite/client/menus/MenuManager.java

```diff
@@ -247,7 +247,8 @@ private void removeNpcOption(NPCComposition composition, String npcOption)
 	@Subscribe
 	public void onMenuOptionClicked(MenuOptionClicked event)
 	{
-		if (event.getMenuAction() != MenuAction.RUNELITE)
+		if (event.getMenuAction() != MenuAction.RUNELITE
+			&& event.getMenuAction() != MenuAction.RUNELITE_PLAYER)
 		{
 			return; // not a player menu
```

Comment:
Adam commented in discord and I agree somewhat that it's odd that this comment isn't updated to reflect the code that this method actually runs. This should mention that player and managed widget menus are handled here.